.github/workflows/build_ukf.yml

```
@@ -0,0 +1,47 @@
+name: Deploy UKFTractography Container
+
+on:
+  push:
+    branches: [ main ]
+    paths: 'containers/ukftractography/*'
+    # Publish semver tags as releases.
+    tags: [ 'v*.*.*' ]
+  pull_request:
+    branches: [ main ]
+    paths: 'containers/ukftractography/*'
+  workflow_dispatch:
+  
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+      # Check out code
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Cache Docker layers
+        uses: actions/cache@v2
+        with:
+          path: /tmp/.buildx-cache
+          key: ${{ runner.os }}-buildx-${{ github.sha }}
+          restore-keys: |
+            ${{ runner.os }}-buildx-
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+      # And make it available for the builds
+      - name: Build and push
+        uses: docker/build-push-action@v2
+        with:
+          context: containers/ukftractography
+          push: true
+          tags: tigrlab/ukf:latest
```

keep docker container name consistent with folder name

```suggestion
          tags: tigrlab/ukftractography:latest
```